open-metadata-implementation/admin-services/admin-services-client/src/main/java/org/odpi/openmetadata/adminservices/client/OMAGServerPlatformConfigurationClient.java

```diff
@@ -278,4 +281,30 @@ public Connection getPlatformSecurityConnection() throws OMAGNotAuthorizedExcept
 
         return restResult.getOMAGServerConfigs();
     }
+    /*
+     * =============================================================
+     * Learn about all possible access services that a platform has
+     */
+
+    /**
+     * Return the list of access services for this platform.
+     *
+     * @return list of access service descriptions registered on this platform
+     * @throws OMAGNotAuthorizedException the supplied userId is not authorized to issue this command.
+     * @throws OMAGInvalidParameterException invalid parameter.
+     * @throws OMAGConfigurationErrorException unusual state in the admin server.
+     */
+    public List<RegisteredOMAGService> getRegisteredAccessServices() throws OMAGNotAuthorizedException,
```

Comment:
I do not see the value of adding this call here -if it is not accompanied by equivalent calls for the other types of registered services - engine, view, integration

Reply:
I have removed it now - I found the PlatformServices Client.